player/command.c

```diff
@@ -1637,10 +1634,10 @@ static int mp_property_ao_volume(void *ctx, struct m_property *prop,
         };
         return M_PROPERTY_OK;
     case M_PROPERTY_PRINT: {
-        ao_control_vol_t vol = {0};
+        float vol;
```

Comment:
missing init

Reply:
We are not using the uninitialized memory, so it shouldn't be fine, or?

Reply:
better safe than sorry if an ao accidentally doesn't write to it.

Reply:
Indeed. Changed.